docs/topics/email.txt

```diff
@@ -51,7 +51,7 @@ are required.
 
 * ``subject``: A string.
 * ``message``: A string.
-* ``from_email``: A string.
+* ``from_email``: A string. If ``None`` uses the DEFAULT_FROM_EMAIL.
```

Comment:
Maybe change to the following to match similar messages below:
```
If ``None``, Django will use the value of the :setting:`DEFAULT_FROM_EMAIL` setting.
```

Reply:
Thanks, done.